.lgtm.yml

```diff
@@ -6,3 +6,7 @@ extraction:
         - python3-six
         - python3-yaml
         - python3-jsonschema
+
+path_classifiers:
+  vendor:
+    - src/bindings/python/utils
```

Comment:
```suggestion
    - src/bindings/python/flux/utils
```

Reply:
Yep, another thing I caught in debugging. :+1: nice catch!